content/Watsonx/WatsonxAI/102.md

```diff
@@ -0,0 +1,250 @@
+---
+title: '102: Parameter config and output formats'
+timeToComplete: 45
+updated: 2023-10-06
+---
+
+<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />
+
+# watsonx.ai L3 part 2: Model parameters and prompting for output formats
+
+This part 2 of the watsonx.ai L3 badge lab. In the previous lab we experiemented with different foundation models and saw how tweaks to a zero shot prompt can make a big difference in the generated output.
+
+In this lab we will explore other techniques for getting the best results from a foundation model.
+
+## Modifying parameters
+
+In this lab we will continue working in the Prompt Lab, and experiment with configurable model parameters. If necessary, [navigate back to the watsonx.ai home screen](/watsonx/watsonxai/000#accessing-watsonxai-from-ibm-cloud), and enter the Prompt Lab UI
```

Comment:
The link is broken.

Reply:
(/watsonx/watsonxai/000#accessing-watsonxai-from-ibm-cloud),
(/watsonx/watsonxai/000#creating-a-new-prompt-lab-session)

Reply:
👍 Thanks, will fix in subsequent PR